server/sonar-web/src/main/js/apps/issues/components/BulkChangeModal.js

```diff
@@ -82,7 +82,7 @@ export default class BulkChangeModal extends React.PureComponent {
     super(props);
     let organization = props.component && props.component.organization;
     if (props.organization && !organization) {
-      organization = props.organization.key;
+      ({ organization } = props);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract the assignment of "{ organization }" from this expression. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AAssignmentWithinCondition)